tests/lib/linter/linter.js

```diff
@@ -4602,6 +4682,19 @@ describe("Linter", () => {
             });
         });
 
+        it("does not include suggestions in autofix results", () => {
```

Comment:
> I think we need to add some tests to verify that suggestions will not be included in formatter output.

@ilyavolodin, is this test what you had in mind?

Reply:
Yes, thanks! That's a good test to have.